DATAFORMATS.md

```diff
@@ -58,6 +58,265 @@ The traffic data expected to be passed in regularly to `ele.updateData(traffic)`
 }
 ```
 
+The following is sample JSON for rending a basic one-to-many graph representation:
+
+```js
+{
+  "renderer": "global",
+  "name": "edge",
+  "nodes": [
+    {
+      "renderer": "region",
+      "name": "INTERNET",
+      "updated": 1466838417510,
+      "nodes": [
+      ],
+      "class": "normal"
+    },
+    {
+      "renderer": "region",
+      "name": "us-east-1",
+      "maxVolume": 50000,
+      "class": "normal",
+      "updated": 1466838546805,
+      "nodes": [
+        {
+          "name": "INTERNET",
+          "class": "normal"
+        },
+        {
+          "name": "alpha",
+          "metadata": {
+            "streaming": true
+          },
+          "class": "normal"
+        },
+        {
+          "name": "bravo",
+          "metadata": {
+            "streaming": true
+          },
+          "class": "normal"
+        },
+        {
+          "name": "charlie",
+          "metadata": {
+            "streaming": true
+          },
+          "class": "normal"
+        },
+        {
+          "name": "delta",
+          "metadata": {
+            "streaming": true
+          },
+          "class": "normal"
+        },
+        {
+          "name": "echo",
+          "metadata": {
+            "streaming": true
+          },
+          "class": "normal"
+        },
+        {
+          "name": "foxtrot",
+          "metadata": {
+            "streaming": true
+          },
+          "class": "normal"
+        }
+      ],
+      "connections": [
+        {
```

Comment:
Remove connections that have no target. I notice that this was just copied from [vizceral-example/src/sample_data_simple.json](https://github.com/Netflix/vizceral-example/blob/master/src/sample_data_simple.json) so I removed those dead connections from there as well. I also removed some extraneous data.